src/messages/IPostMessageSentHandler.ts

```diff
@@ -6,14 +6,14 @@ import { IMessage } from './IMessage';
 export interface IPostMessageSentHandler {
     /**
      * First step when a handler is executed: Enables the handler to signal
-     * to the Rocketlets framework whether the it shall actually execute for the message
+     * to the Rocketlets framework whether handler shall actually execute for the message
      * about to be sent.
      *
      * @param message The message which is being sent
      * @param read An accessor to the environment
      * @return true: run the pre-logic
      */
-    isMessageApplicable(message: IMessage, read: IRead): boolean;
+    checkPostMessageSent(message: IMessage, read: IRead): boolean;
```

Comment:
@graywolf336 the very generic API would be

```
export interface IPreMessageSentHandler {
    /**
     * First step when a handler is executed: Enables the handler to signal
     * to the Rocketlets framework whether handler shall  actually execute for the message
     * about to be sent.
     *
     * @param message The message which is being sent
     * @param read An accessor to the environment
     * @return true: run the pre-logic
     */
    checkPreMessageSent(message: IMessage, read: IRead): boolean;

    executePreMessageSent(message: IMessage, read: IRead, modify: IModify):IExecutionResult;
}
```

However, I am not a big fan of that: Basically, each developer implementing the interface has a very wide scope of possibilities which make it impossible to determine on installation which consequences the Rocketlet installation has.